packages/api/src/controllers/asset.ts

```diff
@@ -1,10 +1,15 @@
 import { authorizer } from "../middleware";
 import { validatePost } from "../middleware";
-import { Request, RequestHandler, Router } from "express";
+import { Request, RequestHandler, Router, Response } from "express";
 import jwt, { JwtPayload } from "jsonwebtoken";
 import { v4 as uuid } from "uuid";
-import mung from "express-mung";
-import tus from "tus-node-server";
+import {
+  Server as TusServer,
+  EVENTS as TUS_EVENTS,
+  Upload as TusUpload,
+} from "@tus/server";
```

Comment:
Alternative:
```suggestion
import * as tus from "@tus/server";
```

then just use `tus.Server`, `tus.EVENTS`, etc